vrpy/README.md

```diff
@@ -0,0 +1,35 @@
+# Hyperheuristic 
```

Comment:
I think most of this is covered in the appropriate places in `hyperheuristics.py` (?)